src/modules/announcements/components/AnnouncementsList.jsx

```diff
@@ -0,0 +1,53 @@
+import ReactStars from "react-rating-stars-component";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
+import styles from "./AnnouncementsList.module.scss";
+
+export const AnnouncementsList = ({ filteredAnnouncements }) => {
+  if (filteredAnnouncements.length === 0) {
+    return (
+      <h3 className={styles.announcementNotFound}>
```

Comment:
Create a component inside this component called `AnnoucementsListEmptyState` and move it there